packages/wonder-blocks-announcer/src/Announcer.ts

```diff
@@ -0,0 +1,257 @@
+// TODO: publish wonder-blocks-style package WB-1776
+// import {srOnly} from "../../wonder-blocks-style/src/styles/a11y";
+
+import {
+    PolitenessLevel,
+    RegionFactory,
+    RegionDictionary,
+    RegionDef,
+} from "../types/Announcer.types";
+
+import {
+    createRegionWrapper,
+    createDuplicateRegions,
+    removeMessage,
+} from "./util/dom";
+import {alternateIndex, debounce} from "./util/util";
+
+const REMOVAL_TIMEOUT_DELAY = 5000;
```

Comment:
It's admittedly a bit of a random number... long enough to let the platform accessibility API read out a message of unknown length before it's removed, and also longer than the default initial timeout. Initially I made this configurable by the user but I removed it due to added complexity to maintain.

Reply:
**idea**: What if we determined the delay dynamically based on the number of words in the announcement multiplied by something like 500ms? That way long messages are accounted for! This might also account for translations that might be longer, though we would be assuming approximately how long each word is read!

Reply:
It's an interesting thought. I'm hesitant to go that route as screen reader speech rates are configurable, so it's a moving target. Perhaps we could mark that down for a future improvement if it comes up in production?